src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1465,4 +1469,36 @@ public static MobEffect loadMobEffect(CompoundTag nbt, String key, @Nullable Mob
         }
     }
 
+    private static final ThreadLocal<RegistryResourceAccess> registryResouceAccess = new ThreadLocal<>();
```

Comment:
Typo in field name; missing a `r` in `Resource`.
```suggestion
    private static final ThreadLocal<RegistryResourceAccess> registryResourceAccess = new ThreadLocal<>();
```